### ISSUE TITLE
USB DRIVE REMOVED , HACK WONT GO AWAY!

### ISSUE THREAD

**Issue by bmckee1989** [pathartl/BleemSync#220]
Some how I've manage to flash what was on my usb stick to the PSClassic.  Even after removal of usb drive, and a default reset it is all still there.  I haven't altered script in anyway!!!  Is this a known issue with 4.1?  I can provide pictures or even video.

**Comment by andshrew** [comment]
Can you confirm you have fully power cycled the system (physically remove the power lead and don't reconnect it until the power LED has gone off).

**Comment by bmckee1989** [comment]
Yes.  i have only just discovered this issue so i am still trying to fully grasp what is happening I made a short video showing the games left behind after USB has been removed numerous boots ago.  None boot though.
https://www.youtube.com/watch?v=8TnDe0o5caM

**Comment by pathartl** [comment]
Have you used any other tool other than BleemSync 0.4.1?

**Comment by bmckee1989** [comment]
No sir, didn't even know there were alt hacks.  only thing i can think of that i did wrong was replace everything instead of just BleemSync folder when i upgraded from 0.3.1 to 0.4.1

**Comment by mikewolfskin** [comment]
After watching your video it looks to me like you have or had a different hack installed called gpghax. As I recall gpghax set up a Custom Games folder and it stays on the system transparent when the USB was removed. gpghax has an uninstall tool that removes it. gpghax uninstall: https://github.com/justMaku/gpghax

**Comment by bmckee1989** [comment]
This will probably work as I tried the first hack available I've had PS classic since day one so I may have actually ran that hack until it posted the redirect to bleemsync.  I am trying it now I just realised the PS1 isn't actually reading my usb flash drive anymore so having to go through my collection and find one working.    This one used to work and now that it isn't that is probably why I am just noticing those three broken items ALWAYS there.
